NuGet.Config

```diff
@@ -6,5 +6,7 @@
     <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
     <add key="cli-deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="myget.org dotnet-buildtools" value="https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
+    <add key="myget.org nugetbuildtasks netcore" value="https://myget.org/F/brthor-netcore-testpackages/api/v3/index.json" />
```

Comment:
I hand rolled a .net core version of https://github.com/nuget/nuGet.BuildTasks and stuck it here. 
@jasonmalinowski Can we work together to come up with a long term plan for how these can be consumed in the cli, using .net core msbuild?